ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/user_management_service.proto

```diff
@@ -98,6 +109,15 @@ message User {
     // Users for participant administrators MAY have an associated primary party.
     // Optional
     string primary_party = 2;
+
+    // When set to 'true' deny this user is denied all access to the Ledger API.
+    // When set 'false' this users can access the Ledger API access at the level corresponding to its user rights.
+    // Optional
```

Comment:
```suggestion
    // When set, then the user is denied all access to the Ledger API.
    // Otherwise, the user has access to the Ledger API as per the user's rights.
    // Optional, Read-Write
```